lib/index.js

```diff
@@ -304,60 +312,71 @@ TradeOfferManager.prototype.getOffer = function(id, callback) {
 	});
 };
 
-TradeOfferManager.prototype.getOffers = function(filter, historicalCutoff, callback) {
+TradeOfferManager.prototype.getOffers = function (filter, historicalCutoff, callback) {
 	if (typeof historicalCutoff === 'function') {
 		callback = historicalCutoff;
 		historicalCutoff = new Date(Date.now() + 31536000000);
 	} else if (!historicalCutoff) {
 		historicalCutoff = new Date(Date.now() + 31536000000);
 	}
 
-	// Currently the GetTradeOffers API doesn't include app_data, so we need to get descriptions from the WebAPI
+	// Currently the GetTradeOffers API doesn't include app_data,
+	// so we need to get descriptions from the WebAPI.
 
 	var options = {
-		"get_sent_offers": 1,
-		"get_received_offers": 1,
-		"get_descriptions": 0,
-		"language": this._language,
-		"active_only": (filter == EOfferFilter.ActiveOnly) ? 1 : 0,
-		"historical_only": (filter == EOfferFilter.HistoricalOnly) ? 1 : 0,
-		"time_historical_cutoff": Math.floor(historicalCutoff.getTime() / 1000)
+		get_sent_offers: 1,
+		get_received_offers: 1,
+		get_descriptions: 0,
+		language: this._language,
+		active_only: (filter === EOfferFilter.ActiveOnly) ? 1 : 0,
+		historical_only: (filter === EOfferFilter.HistoricalOnly) ? 1 : 0,
+		time_historical_cutoff: Math.floor(historicalCutoff.getTime() / 1000),
 	};
 
-	this._apiCall('GET', 'GetTradeOffers', 1, options, (err, body) => {
-		if (err) {
-			callback(err);
+	this._apiCall('GET', 'GetTradeOffers', 1, options, (getTradeOffersErr, body) => {
+		if (getTradeOffersErr) {
+			callback(getTradeOffersErr);
 			return;
 		}
 
 		if (!body.response) {
-			callback(new Error("Malformed API response"));
+			callback(new Error('Malformed API response'));
 			return;
 		}
 
-		// Make sure at least some offers are well-formed. Apparently some offers can be empty just forever. Because Steam.
-		var allOffers = (body.response.trade_offers_sent || []).concat(body.response.trade_offers_received || []);
-		if (allOffers.length > 0 && (allOffers.every(Helpers.offerMalformed) || allOffers.some(Helpers.offerSuperMalformed))) {
-			callback(new Error("Data temporarily unavailable"));
+		// Make sure at least some offers are well-formed.
+		// Apparently some offers can be empty just forever. Because Steam.
+		var allOffers = (body.response.trade_offers_sent || [])
+			.concat(body.response.trade_offers_received || []);
+		if (allOffers.length > 0
+				&& (allOffers.every(Helpers.offerMalformed) || allOffers.some(Helpers.offerSuperMalformed))) {
+			callback(new Error('Data temporarily unavailable'));
```

Comment:
Long lines are much more preferable to me than short-but-unreadable ones. This is getting into that territory of trying to please the machine instead of the reader.